system/assets/js/admin/lib/plugins/jquery.validate.js

```diff
@@ -1385,11 +1385,9 @@ $.extend( $.validator, {
 
 		// http://jqueryvalidation.org/maxlength-method/
 		maxlength: function( value, element, param ) {
-			var newLines = value.match(/\r\n|\n|\r/g), addition = 0;
-			if (newLines != null) {
-				addition = newLines.length;
-			}
-			var length = $.isArray( value ) ? value.length + addition : this.getLength( value, element ) + addition;
+			value = value.replace(/\r\n/g, '\n');
```

Comment:
As per the line below, the value might be an array. We should only do this to the value if it's not an array...